pkg/network/operation.go

```diff
@@ -0,0 +1,20 @@
+package network
```

Comment:
Should be on thyra side

Reply:
And should be tested if it's a reusable pacakage :-)

Reply:
thoses function are only wrappers that initialize a client and call thyra's code.
It allow us to mock the thyra code which are basically rpc calls.
I don't really see much to test here. and it would be quite painfull because it would mean mock http calls to the massa node